Activities/Database/UiPath.Database.Activities.Packaging/UiPath.Database.Activities.Packaging.csproj

```diff
@@ -1,112 +1,125 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="..\Database.build.props" />
-  <PropertyGroup>
-    <TargetFrameworks>net461;net5.0-windows;net5.0</TargetFrameworks>
-    <OutputPath>$(ProjectDir)..\..\Output\Activities\Database\</OutputPath>
-    <VersionSuffix></VersionSuffix>
-    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <Platforms>AnyCPU;x86;x64</Platforms>
-    <!--leave empty for ci to replace with prelease tag-->
-  </PropertyGroup>
+	<Import Project="..\Database.build.props" />
+	<PropertyGroup>
+		<TargetFrameworks>net461;net5.0-windows;net5.0</TargetFrameworks>
+		<OutputPath>..\..\Output\Activities\Database\</OutputPath>
+		<AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+		<Platforms>AnyCPU;x86;x64</Platforms>
+		<!--leave empty for ci to replace with prelease tag-->
+		<VersionSuffix></VersionSuffix>
+		<GeneratePackageOnBuild Condition="'$(Configuration)' == 'Release'">false</GeneratePackageOnBuild>
+		<GeneratePackageOnBuild Condition="'$(Configuration)' == 'Debug'">true</GeneratePackageOnBuild>
+		<PackageOutputPath Condition="'$(Configuration)' == 'Debug'">..\..\Output\Activities\Packages\</PackageOutputPath>
+		<AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+	</PropertyGroup>
 
-  <PropertyGroup>
-    <PackageId>UiPath.Database.Activities</PackageId>
-    <PackageTitle>UiPath.Database.Activities</PackageTitle>
-    <PackageIconUrl>https://download.uipath.com/UI_icon.png?web</PackageIconUrl>
-    <PackageLicenseUrl>https://docs.uipath.com/activities/docs/about-the-database-activities-pack</PackageLicenseUrl>
-    <PackageTags>UiPathActivities</PackageTags>
-    <Description>UiPath Database Activities Pack</Description>
-    <Authors>UiPath</Authors>
-    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
-    <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);AddDlls</TargetsForTfmSpecificBuildOutput>
-    <RunPostBuildEvent>OnBuildSuccess</RunPostBuildEvent>
-  </PropertyGroup>
+	<PropertyGroup>
+		<PackageId>Matei.Database.Activities</PackageId>
```

Comment:
Remove this change in package name. It will break compatibility with older versions of the package.